modules/db/src/main/java/org/mule/module/db/internal/domain/connection/DefaultDbConnection.java

```diff
@@ -393,22 +404,52 @@ public int getNetworkTimeout() throws SQLException
     @Override
     protected void resolveLobs(String typeName, Object[] attributes) throws SQLException
     {
-        try (ResultSet resultSet = this.getMetaData().getAttributes(this.getCatalog(), null, typeName, null))
+        Map<Integer, List> dataTypes = getDataTypes(typeName);
```

Comment:
do not use a list and use ResolvedDbType for the values (it exists in module-db).